### DISCUSSION TITLE
Append setting Panel in a modal

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#3777] [Q&A] [answered]
Hi @artf  Can you please help me i want to show the settings panel in my custom modal

**Comment by Abhisheknanda1344463** [comment, marked as answer]
I've achieved it.

**Reply by codingmachine16** [reply]
How???

**Reply by pakhi96** [reply]
Hi @Abhisheknanda1344463 
I've been trying to achieve the same but I'm seeing some errors. Can you please share how you've achieved it?

**Reply by Abhisheknanda1344463** [reply]
hi @pakhi96  @codingmachine16 
i've implemented this way
This is div of modal
```<div class="panel-shortcuts" style= "display:none;" id="panel-shortcuts"></div> ```
```
 var infoContainer = document.getElementById('panel-shortcuts');
 cmdm.add('open-info', function() {
      infoContainer.style.display = 'block';
      modal.setTitle('Settings');
      infoContainer.append(editor.Traits.render());
      modal.setContent(infoContainer);
      modal.open();
    })
```

**Reply by Abhisheknanda1344463** [reply]
@artf Is this the right way to do the same?

**Reply by artf** [reply]
Yeah sure
